### ISSUE TITLE
Build scripts don't run games

### ISSUE THREAD

**Issue by josefnpat** [josefnpat/vapor#98]
When `love.exe` is combined with `vapor_<hash>.love`, it creates `vapor_<hash>.exe`.

Using `vapor_<hash>.exe` with arguments simply runs the combined `vapor_<hash>.love` instead of the argument.

**Comment by qaisjp** [comment]
What? Running vapor_hash.exe with arguments will just run the normal vapor_hash.love with arguments sent to "arg" in love.load

**Comment by josefnpat** [comment]
After using the `build.sh` to create `vapor_win-<arch>-[<hash>].zip`, there is a combined binary of `love.exe` and `vapor_<hash>.love`, named `vapor_<hash>.exe`.

Due to this line;

https://github.com/josefnpat/vapor/blob/master/src/main.lua#L105

With this line, vapor determines where the binary that runs vapor is. When the binary is itself, it calls;

`vapor_<hash>.exe <gamename>_<timestamp>.love`. Because the because the `love.exe` has the `.love` combined to itself, it skips passing the argument to the boot.lua, and runs itself instead.

The result is every time you start a game in vapor in the `build.sh` version of vapor, it runs vapor.

![xhibit](http://jamiedubs.com/fuckflickr/data/xzibit-yo-dawg/web/xzibit-wtf.jpg)

**Comment by qaisjp** [comment]
I honestly don't get it completely, but I think I came across this problem myself whilst testing a compiled exe with a .love in mac. Here's my code if it matters:

<snip>

This is probably completely useless but whatever

**Comment by josefnpat** [comment]
@qaisjp Yes, it is, sorry. The problem isn't reading the argument and determining if it's a valid file, but convincing the `boot.lua` in love to fire up the argument as the game, as opposed to the fused binary.

**Comment by qaisjp** [comment]
modify run.lua or modify the boot.lua? im sure there was something in the lua docs or code that allowed you to change the boot code using require"boot" or smth

**Comment by jadebrew** [comment]
haha dat reaction image.
